packages/components/src/components/table/table-cell/table-cell-styles.ts

```diff
@@ -1,17 +1,20 @@
 import { getCss } from '../../../utils';
-import { addImportantToEachRule, pxToRemWithUnit, getThemedColors, hostHiddenStyles } from '../../../styles';
+import { addImportantToEachRule, getThemedColors, hostHiddenStyles } from '../../../styles';
+import { spacingFluidSmall } from '@porsche-design-system/utilities-v2';
+import { cssVariableTableBorderColor } from '../table/table-styles';
 
 export const getComponentCss = (multiline: boolean): string => {
   return getCss({
     '@global': {
       ':host': {
         ...addImportantToEachRule({
           display: 'table-cell',
-          padding: pxToRemWithUnit(12),
+          padding: spacingFluidSmall,
           margin: 0,
-          borderBottom: `1px solid ${getThemedColors('light').contrastLowColor}`,
+          borderBottom: `1px solid var(${cssVariableTableBorderColor}, ${getThemedColors('light').contrastLowColor})`,
```

Comment:
fallback ${getThemedColors('light').contrastLowColor} is not needed since we are ensuring on our own that correpsonding css variable is always set.

Reply:
Done